src/en/administration/first_steps.md

```diff
@@ -30,6 +30,8 @@ To be included in the list of Lemmy instances on [join-lemmy.org](https://join-l
 
 - [x] Federate with at least one instance from the list
 - [x] Have a site description and icon
+- [x] Have open sign-ups
+- [x] Have at least 5 monthly active users
```

Comment:
This means local users who posted or commented at least once in the past month

Reply:
Added the clarification